arch/arm/soc/st_stm32/common/Kconfig.defconfig.series

```diff
@@ -51,14 +51,14 @@ config PWM_STM32
 
 endif # PWM
 
-if SPI && (SPI_1 || SPI_2 || SPI_3)
+if SPI && (SPI_1 || SPI_2 || SPI_3 || SPI_4 || SPI_5 || SPI_6)
```

Comment:
While this change is good for bug fixing, I wonder if we should just remove the instance check, like this is done for other drivers with instances (UART, PWM) for consistency.
Also, this will be obsolete once we manage to progress with DTS based generation (which might take some time though).

Reply:
I removed the instances for consistency.